changelog/@unreleased/pr-2135.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
```

Comment:
might as well call this a break and major rev. version numbers are inexpensive :-)